models.py

```diff
@@ -141,9 +141,9 @@ class PayMovies(db.Model):
     aspect = db.Column(db.Boolean, nullable=False)
     payenddt = db.Column(db.String(19), nullable=False)
     ds_dist = db.Column(db.Boolean, nullable=False)
-    ds_mov_id = db.Column(db.Integer)
+    ds_mov_id = db.Column(db.Boolean, nullable=False)
```

Comment:
This should be an integer - `dsdist` is a boolean specifying its existence, where `dsmovid` specifies what movie ID it should download to package and send to the DS. We may want to better document how each field is used somewhere in source as well, alongside the Wii no Ma docs repository.

Reply:
ah I'll fix it

Reply:
Fixed